README.md

```diff
@@ -1,6 +1,10 @@
-# Scaffold for social media app with Ruby on Rails
+# Social Media Website
 
-> This repo includes intial code for social media app with basic styling. Its purpose is to be a starting point for Microverse students.
+> This is a project to simulate a social media website with all its functionality
+
+![social_media_gif](https://media1.tenor.com/images/a8e963f6d83f61bf82c90f5a8c2a26b5/tenor.gif?itemid=15614780)
```

Comment:
Don't you think an actual screenshot from your app will be more appropriate for display here?